src/modules/languages/python.nix

```diff
@@ -62,8 +62,8 @@ let
         [ -f "${config.env.DEVENV_STATE}/poetry.lock.checksum" ] && rm ${config.env.DEVENV_STATE}/poetry.lock.checksum
       ''}
       ${if cfg.uv.enable then ''
-        echo uv venv "$VENV_PATH"
-        uv venv "$VENV_PATH"
+        echo uv venv -p ${cfg.package.interpreter} "$VENV_PATH"
```

Comment:
Should it be package.interpeter?

Reply:
Thanks for the pointer. And yes, it should be ``package.interpreter``